### DISCUSSION TITLE
Developer Needed from the Community

### DISCUSSION THREAD

**Discussion by umtcylmz** [ant-media/Ant-Media-Server#5493] [Announcements]
Hi everyone,

We have a customer who is looking for a developer to create a live-streaming application for him. 

Who to contact: Nur Çakır - nur.cakir@antmedia.io

Here below you can check the project's details:

- Live broadcast
- Dual Live Broadcast (Challenge Competition)
- Screen sharing
- Broadcast Recording
- Gift System (Gift sending animation set, control of the amount of gift sent to the user and log records of these transactions in the admin panel.)
- Integration with publisher / viewer website.
- Adjust video quality (480P, 720P, 1080P, 2K, 4K) Broadcast
- Broadcast chat system
- Chat control management while the broadcaster is live (Banning with partial authorization in slang or aggressive chat correspondence)
- Publisher chat message on/off feature
- Publisher voting system

If you are interested please get back to us under this message or directly contact nur.cakir@antmedia.io.

 Thank you

**Comment by umtcylmz** [comment]
We have found a developer for this request.

**Comment by hemantbhr** [comment]
@umtcylmz Please add me in group where we can direct connect with clients i have implemented ANT Media in my last 7 Projects this includes Canvas Streaming ,Live Streaming , Multi Streaming and Live commerce . if anyone need then can be contacted on hemant.bohra1987@gmail.com

**Reply by mekya** [reply]
Thank you @hemantbhr , could you please let us know the works you've implemented with Ant Media? You can either send here to make it visible or send to contact@antmedia.io if they are confidential.
